docs/manual/docs/user-guide/associating-resources/using-filestore.md

```diff
@@ -25,4 +25,15 @@ A file uploaded in this way will be exported in the metadata export file (MEF).
 
 ## Filestore configuration
 
-By default, the maximum file size is set to 100Mb. This limit is set in `/services/src/main/resources/config-spring-geonetwork.xml` with the parameter `maxUploadSize`.
+By default, the maximum file size for attachment uploads is set to 100Mb. When building GeoNetwork from source, this limit can be changed in `/services/src/main/resources/config-spring-geonetwork.xml` with the parameter `maxUploadSize`.
+
+For deployment on an application server, or for adjusting the limit in an already running web application, the value for `maxUploadSize` can be changed in `WEB-INF/config.xml` accordingly:
```

Comment:
I don't think this is correct, this value is used for the old Jeeves MVC services, while the `/services/src/main/resources/config-spring-geonetwork.xml` value is used for Spring MVC services.

It is not clear to me if we still have Jeeves MVC services for file uploads or they have all been migrated to Spring MVC.

Reply:
I just worked on the following PR #8427 where I tested the file upload size error handling so I changed the upload size several times during testing and I would set it by adding the following option to the **CATALINA_OPTS** on application startup.

`-Dapi.params.maxUploadSize=1000000000`

The value is in bytes. So this sample sets the upload size to 1GB.

That value is used here
https://github.com/geonetwork/core-geonetwork/blob/9afd1bfee2d093006ae94724ee5210ec1d4cc40b/services/src/main/resources/config-spring-geonetwork.xml#L82
And seems to default to default to 100 MB. 

https://github.com/geonetwork/core-geonetwork/blob/9afd1bfee2d093006ae94724ee5210ec1d4cc40b/services/src/main/webapp/WEB-INF/config.properties#L21

I believe the maxUploadSize specified by Jeeves may be used for something else?  I have not tested that value so I'm not sure where it is used?

Reply:
Thanks @ianwallen for your clarification. I should have taken a more thorough look into the current configuration possibilities. I will revert my misleading lines in the proposed commit and instead just include one sentence on how you suggested to configure the `maxUploadSize` during application deployment.